.travis.yml

```diff
@@ -35,8 +28,6 @@ env:
         - PYTHON_VERSION=3.6 PYTEST_VERSION=3.1
         - PYTHON_VERSION=3.6 PYTEST_VERSION=3.2
 
-matrix:
```

Comment:
Actually I think it's nicer to do the other way around. The matrix in env would actually run for all array elements for the settings above (e.g. if you had arrays for e.g. ``os`` or ``language``, etc), but these above look like were set up with one setting in mind.
(see e.g the how the travis.yml in astropy-helpers translate to an actual matrix on travis.org).

Reply:
In that case do I need to be explicit about `os` and `env` for each row in the matrix then? It seems more concise just to list the combinations of variables that I want to test for linux, and then only have to be more explicit for the osx case.

Reply:
No, only the ones that are different from the global ones. E.g. os will be linux unless you state otherwise. 
But yes, you need to say env when it's changing something from the global ones.

Actually it could stay as it were before, this way the osx job is not being picked up